docs/MAASUI.md

```diff
@@ -176,6 +176,20 @@ You should avoid using \`TSFixMe\` unless you really get stuck.
 
 As a general rule, we concentrate on user-centric testing and avoid testing implementation details. For that reason usage of test attributes such as `data-testid` should be avoided. Any occurrence of such will usually be for historical reasons.
 
+#### Vitest and testing-library
+
+Vitest is the native testing framework for Vite. The Vitest API is (mostly) a drop-in replacement for Jest, which we used in the past. We use Vitest for our integration/unit tests (files that end in `.test.tsx`). By default, when running these tests, Vitest will enter "watch" mode - as soon as file changes are detected, the test(s) will automatically re-run.
+
+React Testing Library is a package from the `testing-library` collection - it's a library of functions that help us test our React code. React Testing Library focuses on accessible testing, and discourages testing implementation details (internal component state, component lifecycle functions etc.). To this end, it renders the React code into actual DOM nodes, as opposed to libraries like Enzyme (the previous standard for testing React) which render the React DOM. Components should be accessed through accessible attributes such as roles, names, and labels.
```

Comment:
```suggestion
[React Testing Library](https://testing-library.com/docs/) is our primary tool for testing React components. It encourages testing user interactions rather than implementation details (internal component state, component lifecycle functions etc.).

To this end, it renders the React code into actual DOM nodes, as opposed to libraries like Enzyme (the previous standard for testing React) which render the React DOM. Components should be accessed through accessible attributes such as roles, names, and labels.
```